### DISCUSSION TITLE
doris并发批量插入问题

### DISCUSSION THREAD

**Discussion by linlianghong** [apache/doris#19336] [RFC]
**背景**
spring程序，mybatisplus操作框架，每秒3000TPS的数据量对doris进行插入，线程池分配了20个线程，每次批量插入500条（连接URL已加rewriteBatchedStatements）
**问题描述**
前3分钟的插入效率正常，能基本3S内的延迟，但是在3分钟之后插入性能越来越慢，从原本的1S插入500条到后面需要17S，效率呈指数下降，下图insert对应的耗时为单批insert into 500条数据的耗时，对标切换成mysql数据库是正常并且稳定插入耗时的
![image](https://user-images.githubusercontent.com/41034640/236609962-c46da723-dbcb-4807-a2d0-b514c9f31680.png)
**问**
是doris有什么参数需要调整么？

**Comment by yagagagaga** [comment]
It's not recommend to using JDBC Insert, StreamLoad is the prefer way to load data. Doris is a high performance MPP Database, while it is total different with MySQL, which is a RDBMS. If you insert a few data every time, it will cause Doris to frequently trigger compaction. That's why your Doris will slow down over time.

**Reply by linlianghong** [reply]
Thank you very much
